app/locations/none.js

```diff
@@ -0,0 +1,37 @@
+import Ember from 'ember';
+
+const {
+  computed: { reads },
+  inject: { service },
+  get
+} = Ember;
+
+const TEMPORARY_REDIRECT_CODE = 307;
```

Comment:
302 is `FOUND`, whereas `307` is `TEMPORARY REDIRECT`, which I'd 307 is more semantically correct. The following note is from [the 302 spec][302spec] (emphasis mine):

> Note: RFC 1945 and RFC 2068 specify that the client is not allowed to change the method on the redirected request.  However, most existing user agent implementations treat 302 as if it were a 303 response, performing a GET on the Location field-value regardless of the original request method. **The status codes 303 and 307 have been added for servers that wish to make unambiguously clear which kind of reaction is expected of the client.**

302 has ambiguity in its handling, whereas 307 is clear
[302spec]: http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.3.3